yt_dlp/YoutubeDL.py

```diff
@@ -4085,17 +4084,6 @@ def get_encoding(stream):
         if plugin_dirs:
             write_debug(f'Plugin directories: {plugin_dirs}')
 
-        # Not implemented
```

Comment:
maybe also add the deprecation warning back?
```diff
diff --git a/yt_dlp/YoutubeDL.py b/yt_dlp/YoutubeDL.py
index b67b68886..5676a01c0 100644
--- a/yt_dlp/YoutubeDL.py
+++ b/yt_dlp/YoutubeDL.py
@@ -551,7 +551,7 @@ class YoutubeDL:
                        (requires writesubtitles or writeautomaticsub)
     include_ads:       - Doesn't work
                        Download ads as well
-    call_home:         - Not implemented
+    call_home:         - Deprecated
                        Boolean, true iff we are allowed to contact the
                        yt-dlp servers for debugging.
     post_hooks:        - Register a custom postprocessor
diff --git a/yt_dlp/__init__.py b/yt_dlp/__init__.py
index f598b6c2f..f88b49efa 100644
--- a/yt_dlp/__init__.py
+++ b/yt_dlp/__init__.py
@@ -556,8 +556,8 @@ def report_deprecation(val, old, new=None):
 
     report_deprecation(opts.sponskrub, '--sponskrub', '--sponsorblock-mark or --sponsorblock-remove')
     report_deprecation(not opts.prefer_ffmpeg, '--prefer-avconv', 'ffmpeg')
+    report_deprecation(opts.call_home, '--call-home')
     # report_deprecation(opts.include_ads, '--include-ads')  # We may re-implement this in future
-    # report_deprecation(opts.call_home, '--call-home')  # We may re-implement this in future
     # report_deprecation(opts.writeannotations, '--write-annotations')  # It's just that no website has it
 
     # Dependent options
```

Reply:
nah, the CLI opts need to remain so that youtube-dl configs can be ported to yt-dlp

Reply:
this is also forgetting about `--no-call-home`